package.json

```diff
@@ -15,6 +15,8 @@
     "build": "node build/build.js",
     "commitmsg": "commitlint -e $GIT_PARAMS"
   },
+  "main": "src/index.js",
+  "module": "src/index.js",
```

Comment:
Should one or both of these point to a built bundle instead of a source file?

Reply:
The main property usually points at a built bundle, but we don't actually publish one.  If we ever do so, we can make the change then.